lib/LedgerSMB/Scripts/login.pm

```diff
@@ -18,6 +18,8 @@ This script contains the request handlers for logging in and out of LedgerSMB.
 
 package LedgerSMB::Scripts::login;
 
+use version 'v5.10.0';
```

Comment:
Thanks for the contributions. A question regarding this change: In light of the article referenced below, should we discuss this change on the mailing list?

https://www.effectiveperlprogramming.com/2010/07/watch-out-for-side-effects-with-use-version/